.claude/skills/archon-cli/manage-run/manage-runs.md

```
@@ -0,0 +1,127 @@
+tier keywords or aliases at launch — the persistent config stays untouched:
+
+```bash
+# Rebind one or more tiers / @aliases for this run only; repeat per binding
+archon workflow run archon-fix --branch fix/x "..." \
+  --model large=pi/minimax/minimax-m3 \
+  --model small=openai/gpt-5-mini
+```
+
+Spec format: `<tier|@alias>=<provider>/<model-ref>` where `<model-ref>` is what
+the named provider expects (for Pi backends that is itself a `<vendor>/<model>
+ref, e.g. `pi/minimax/minimax-m3`). An unknown tier/alias errors with the list
```

markdownlint-cli2 (0.23.2)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)